src/components/skills-quiz/SearchCourseCard.jsx

```diff
@@ -106,7 +107,13 @@ const SearchCourseCard = ({ index }) => {
 
   return (
     <div>
-      <h3 className="mb-3">Get started with these courses</h3>
+      <h3 className="mb-3">
+        <FormattedMessage
+          id="enterprise.skills.quiz.v1.course.card.heading"
+          defaultMessage="Get started with these courses"
+          description="Course card heading on skills quiz v1 page"
```

Comment:
I think this is the heading for all the courses rendered below? Can we please update this? Currently from the description, it looks like that the `Get started with these courses` is a heading for a course card?